modules/installing-aws-load-balancer-operator-cli.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator-cli_{context}"]
+= Installing the AWS Load Balancer Operator with the command line
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* Your cluster is configured with AWS as the platform type and cloud provider.
+
+* You are logged into the command line client `oc`.
+
+.Procedure
+
+. Create the `aws-load-balancer-operator` namespace:
++
+[source,terminal]
+----
+oc apply -f namespace.yaml
```

Comment:
These commands look to be out of order. This is trying to apply a file to the cluster that hasn't been created yet. Also, the `cat` command doesn't actually do anything other than output the contents of a file that already exists.

We have a lot of examples in the repo with procedures that create a file with certain contents, and then apply the file to the cluster.

There are lots of ways to do it, but the idea should be the same that the file is created first, then applied to the cluster. 

Here are a few examples you could model after:

* https://docs.openshift.com/container-platform/4.15/security/cert_manager_operator/cert-manager-operator-issuer-acme.html#cert-manager-acme-http01_cert-manager-operator-issuer-acme
* https://docs.openshift.com/container-platform/4.15/networking/metallb/about-advertising-ipaddresspool.html
* https://docs.openshift.com/container-platform/4.15/logging/log_storage/installing-log-storage.html

So recommend looking at these and reworking these steps to follow something similar to one of these.